src/sys/unix/ready.rs

```diff
@@ -402,6 +402,12 @@ impl From<UnixReady> for Ready {
     }
 }
 
+impl From<UnixReady> for Interests {
```

Comment:
Why is this needed?

Reply:
I suggest add platform-specific readiness in `Interests` to support Unix
And conversion from `Interests` to `Ready` to simplify code and get rid of `Interests::from_usize` and `Interests::to_usize`
Since `Interests` should be the subset of `Ready` and never be `empty`

Reply:
Ideally, we can avoid implementing the trait. We can add a `pub(crate)` fn for the conversion. Implementing the `From` trait makes the conversion part of the public API.